kubejs/server_scripts/base/tags/entity_type/enigmatica/mob_spawner_blacklist.js

```diff
@@ -0,0 +1,23 @@
+ServerEvents.tags('entity_type', (event) => {
+    let entities = [
```

Comment:
The Cave Creeper should probably be in here, no?

Reply:
I was just going to change their loot table entirely.